submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/model/FilesSource.kt

```diff
@@ -15,8 +14,26 @@ interface FilesSource {
     fun readText(filePath: String): String
 }
 
-class ListFilesSource(private val files: List<ResourceFile>) : FilesSource {
+class MixedFilesSource(
```

Comment:
this one should be the generic approach so it looks ok for be to be named `FilesSourc`

Reply:
Probably not, because in the attachment process, it's fine to have just `ListFileSource` one. I think it's fine that we have the general `FilesSource` interface.

Maybe this name can be improved? `FallbackFilesSource` or something like that?

Reply:
`it's fine to have just ListFileSource one`  I understand that is fine but why have different behavior based on request type,  in this case it does not represent any extra effort so I think is better to have single approach as is simpler.